kola/tests/podman/podman.go

```diff
@@ -212,7 +212,7 @@ func podmanWorkflow(c cluster.TestCluster) {
 		cmd = fmt.Sprintf("sudo podman images | grep %s", imageID)
 		out, err := c.SSH(m, cmd)
 		if err == nil {
-			c.Fatalf("Image should be deleted but found %s", string(out))
+			c.Fatalf("Image should be deleted but found %s", string(out)[0:12])
```

Comment:
Why trim the image name?

Reply:
I went on autopilot there a bit.  I'll remove it.